manifests/legacy/cron_remover.pp

```diff
@@ -0,0 +1,26 @@
+# @summary Ensure old vaccum and pg_repack crons are purged 
+#
+# If someone upgrades from an ancient v0.x version of the pe_databases module to 2.0 or newer, 
+# the old cron jobs running vaccuum full will not be cleaned up. This can result in a deadlock 
+# when both pg_repack and vacuum full attempt to update a table:
+#
+# See https://tickets.puppetlabs.com/browse/SUP-2571
```

Comment:
This is an internal Puppet link in public code. Non Puppet employees cannot access the link so please remove it from the comment.